### PR TITLE
travis

### PR DESCRIPTION
**Pull request by imaGecko**
travis

### CODE REVIEW

**Comment by Haganeko** [comment]
Doesn't Dismember also need an update?

**Comment by imaGecko** [comment]
if they made a change since

File: game/scripts/npc/abilities/pudge_dismember.txt [hunk 1]
```diff
@@ -59,6 +59,11 @@
         "var_type"        "FIELD_INTEGER"
         "pull_distance_limit"   "125"
       }
+      "06"
+      {
+        "var_type"        "FIELD_INTEGER"
```
Comment (Haganeko) [inline game/scripts/npc/abilities/pudge_dismember.txt]: Should probably be a float.

**Comment by imaGecko** [comment]
#justvalvethings

**Pull request merged**